composer.json

```diff
@@ -14,30 +14,36 @@
         "spryker/glossary": "^3.0.0",
         "spryker/guzzle": "^2.1.0",
         "spryker/kernel": "^3.33.0",
-        "spryker/oms": "^6.0.0 || ^7.0.0 || ^8.0.0 || ^10.0.0 || ^11.0.0",
+        "spryker/oms": "^7.3.0 || ^8.0.0 || ^10.0.0 || ^11.0.0",
         "spryker/propel-orm": "^1.6.0",
         "spryker/quote": "^2.14.0",
-        "spryker/refund": "^4.0.0 || ^5.0.0",
-        "spryker/sales": "^5.0.0 || ^6.0.0 || ^7.0.0 || ^8.0.0 || ^10.0.0 || ^11.0.0",
+        "spryker/refund": "^5.1.0",
+        "spryker/sales": "^8.2.1 || ^10.0.0 || ^11.0.0",
         "spryker/shipment": "^6.0.0 || ^8.0.0",
+        "spryker/silex": "^2.2.0",
         "spryker/step-engine": "^3.0.0",
         "spryker/store": "^1.1.0",
         "spryker/symfony": "^3.1.1",
         "spryker/util-encoding": "^2.0.0",
-        "spryker/zed-request": "^3.2.0"
+        "spryker/zed-request": "^3.2.0",
+        "symfony/event-dispatcher-contracts": "^2.5.0",
+        "symfony/string": "^5.4.0"
     },
     "require-dev": {
-        "phpstan/phpstan": "*",
+        "phpstan/phpstan": "~1.2.0",
```

Comment:
```suggestion
        "phpstan/phpstan": "^1.2.0",
```

Reply:
Both is fine for now. But sure, semver is cool

Reply:
Fixed.